code/modules/research/xenoarchaeology/tools/tools_pickaxe_vr.dm

```diff
@@ -0,0 +1,126 @@
+
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+// Excavation pickaxes - sorted in order of delicacy. Players will have to choose the right one for each part of excavation.
+
+/obj/item/weapon/pickaxe/brush
```

Comment:
Are these all complete copy/pastes of tools_pickaxe.dm  but with the icon file changed?